### PR TITLE
Lifesavers: A Medical Expansion

### PR DESCRIPTION
**Pull request by CosmicPhoenix**
## About The Pull Request
See #163 

## Why It's Good For The Game
Oh god just work please

## Changelog
:cl:
add: In accordance with recent changes to Space Law we have introduced a number of reforms in our Medical and Security departments. ~NT
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Third time's the charm

File: tgstation.dme [hunk 1]
```diff
@@ -600,40 +600,38 @@
 #include "code\FulpstationCode\fulp_world.dm"
 #include "code\FulpstationCode\t5_parts.dm"
 #include "code\FulpstationCode\wintercoats.dm"
-#include "code\FulpstationCode\halloween_costumes\centaur.dm"
```
Comment (ghost) [inline tgstation.dme:603]: Your DMI's out of date bro

load the DME with DM and it'll automatically fix most of it, then use github's compare to manually tick/untick any remaining differences

Comment (CosmicPhoenix) [reply tgstation.dme:603]: It looks like they're all still in the .dme they were just organized alphabetically

Comment (ghost) [reply tgstation.dme:603]: That seems to me like it's gonna cause conflict issues.

Can you manually roll it back and then re-tick the relevant files?

You could just copy your master .dme in here if that works

Comment (CosmicPhoenix) [reply tgstation.dme:603]: Yeah, I'll try to mess around with it

**Comment by CosmicPhoenix** [comment]
Tfw you go through and unedit the lockers because you don't have to edit them but you still get merge conflicts. Time to figure out Git I suppose...

**Pull request closed without merging**